pkgs/applications/misc/authy/default.nix

```diff
@@ -0,0 +1,119 @@
+{ alsaLib, at-spi2-atk, at-spi2-core, atk, autoPatchelfHook, cairo, cups
+, dbus, electron_9, expat, fetchurl, gdk-pixbuf, glib, gtk3, lib
+, libappindicator-gtk3, libdbusmenu-gtk3, libuuid, makeWrapper
+, nspr, nss, pango, squashfsTools, stdenv, systemd, xorg
+}:
+
+let
+  # Currently only works with electron 9
+  electron = electron_9;
+  pname = "authy";
+  version = "1.8.3";
+  rev = "5";
+
+  buildInputs = [
+    alsaLib
+    at-spi2-atk
+    at-spi2-core
+    atk
+    cairo
+    cups
+    dbus
+    expat
+    gdk-pixbuf
+    glib
+    gtk3
+    libappindicator-gtk3
+    libdbusmenu-gtk3
+    libuuid
+    nspr
+    nss
+    pango
+    stdenv.cc.cc
+    systemd
+    xorg.libX11
+    xorg.libXScrnSaver
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXtst
+    xorg.libxcb
+  ];
+
+in
+
+stdenv.mkDerivation {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
I remember reading over on NixOS Discourse that using `rec` is an anti-pattern and there's also [Nix.dev](https://nix.dev/anti-patterns/language.html#rec-expression) which suggests this. Is using `rec` better in some way?

Reply:
There are other people with other opinions but the default mkDerivation uses rec and if used in the default way it is not an anti pattern.

Reply:
okay, thanks. I'll make the changes.